cores/esp8266/umm_malloc/umm_malloc.cpp

```diff
@@ -1610,9 +1764,12 @@ static void *_umm_realloc( void *ptr, size_t size ) {
     /* And don't forget to adjust the pointer to the new block location! */
 
     ptr    = (void *)&UMM_DATA(c);
+
+    /* Now resume critical section... */
+    UMM_CRITICAL_ENTRY(id_realloc);
   }
 
-  /* Now calculate the block size again...and we'll have three cases */
+  // Now calculate the block size again...and we'll have three cases /
```

Comment:
We're compiling this as. cpp, but upstream is still. c. Please keep original comment style.

Reply:
Huh, Don't know how that happened. And I missed it with all the diffs that I do to remove accidents.